frontend/src/components/lib/Gallery/index.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import styles from './Gallery.module.scss';
 import Button from '../Button';
```

Comment:
```suggestion
import Button from '@/components/lib/Button';
```

Utilise le `@` pour les chemins